packages/components/shared/src/mixins/form-control.ts

```diff
@@ -78,6 +80,7 @@ export function FormControlMixin<T extends Constructor<ReactiveElement>>(
       }
     }
 
+    /** Native labels property. */
```

Comment:
Maybe we should add here `@type NodeListOf<HTMLLabelElement> | null ...` as well?
Because it's not rendering properly:

<img width="1063" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/02672318-ae7d-44b3-9109-4ed29225c19d">